src/pre.js

```diff
@@ -22,11 +22,12 @@ async function emitTelemetry() {
   // All variables we need from the runtime are set in the Deployment constructor
   const deployment = new Deployment()
   const telemetryUrl = `${deployment.githubApiUrl}/repos/${deployment.repositoryNwo}/pages/telemetry`
-  core.info(`Sending telemetry for run id ${deployment.workflowRun}`)
+  const conclusion = core.getInput('conclusion') || null
```

Comment:
Do we need the `|| null` here? I thought that may be the implicit default with JavaScript. @JamesMGreene Would probably know better than me though!

Reply:
Oh, it's because the empty string `""` is falsy in javascript and truthy in ruby, so on the api side we want `data["conclusion"]` to have the correct truthiness (technically `data["conclusion"].presence` does this for us, but I wanted to be explicit).

Reply:
`undefined` is the implicit default in JavaScript but `core.getInput(...)` in particular will return an empty string by default, so converting those into `null` values here makes sense to me. 👍🏻